src/components/Ui/Dropdown.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div class="position-relative">
+    <button @click="open = !open" class="button mr-1">
+      <Icon name="threedots" size="25" class="v-align-text-bottom" />
```

Comment:
The threedots icon should not be part of the component, we should be able to use that component with others icons. Maybe we can simply use `<slot/>` for define the clickable content.